docs/product-repo-plan-proposal.md

```diff
@@ -0,0 +1,56 @@
+# Enabling Performance Testing in the Product Repos
+
+## Value Statement
+
+We want performance testing to be consistent across all product repos that we test.
+All repos (in particular, coreclr and corefx) should run in the same way with the same benchmarks so that we can easily determine where a regression occured.
+
+## Promises
+
+- A consistent AzDO yaml template in arcade for performance testing that can be used across the product repositories with minimal set up for product repo owners.
+- A helix proj file that runs all of the benchmarks and splits them in a sane way
+
+## Success Metric(s)
+
+- Product repositories use the provided yaml template to run all performance testing
+- Product repositories do not try to skirt the yaml template to run performance testing only on the tests they care about
+- Setup time for adding performance testing is as simple as adding a new job that references the yaml template that we provide
+
+## Procedural
+
+- Combine the yaml we already use and, likely, the yaml that has been worked on in coreclr (why reinvent the wheel), with any additional logic necessary for pulling down and running the performance repository in product repos
+  - yaml will need to include any additional parameters/knobs that the product repos may want to turn (ie tiered/non-tiered/minopts, pgo/no-pgo, etc)
```

Comment:
Yes - I like this.  It brings up a question for me though.  How are we going to generalize taking parameters like this and (1) applying them, and (2) making sure that they impact the name/config associated with the results so that it's clear what the config was when looking at results?